python/tvm/relay/op/nn/nn.py

```diff
@@ -1777,3 +1777,22 @@ def cross_entropy(predictions, targets):
       The computed result.
     """
     return _make.cross_entropy(predictions, targets)
+
+
+def cross_entropy_with_logits(predictions, targets):
```

Comment:
Can we name this `cross_entropy_from_logits` everywhere instead

Reply:
Actually, on second thought, can we just add a kwarg for `cross_entropy` for `from_logits`?

Reply:
It is a name from tensorflow, is there a reason to change the name?
I also do not think it should be a kwargs - it look like adding two functionality to one function, which is a common software antipratice.

Reply:
I don't feel too strongly about it either way; I've been using keras interface a lot so I was thinking about keras' declaration:

```
keras.losses.categorical_crossentropy(y_true, y_pred, from_logits=False, label_smoothing=0)
```